components/WeeklyOverview/index.js

```diff
@@ -3,14 +3,16 @@ import styled from 'styled-components';
 import { dummyMedications } from '../../dummydata';
 import { uid } from 'uid';
 
+const hoursOfDay = Array.from({ length: 24 }, (_, i) => i);
```

Comment:
please use more descriptive names for your parameters:

```suggestion
const hoursOfDay = Array.from({ length: 24 }, (_, index) => index);
```